Dockerfile

```diff
@@ -10,17 +10,21 @@ RUN apt-get update && apt-get install -y \
     curl \
     && rm -rf /var/lib/apt/lists/*
 
+COPY --from=ghcr.io/astral-sh/uv:0.5.5 /uv /uvx /bin/
+
+ENV UV_SYSTEM_PYTHON=1
+
 # Install PyTorch with CUDA support
-RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+RUN uv pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Important change here is we will be fetching CPU only wheels instead of CUDA ones, since there's another Dockerfile specifically for CUDA